tests/test_ebs.py

```diff
@@ -1185,3 +1187,36 @@ def test_filter_match(self):
             resources = p.run()
         self.assertEqual(len(resources), 1)
         self.assertEqual(resources[0]["VolumeId"], "vol-0c7930681fe5a17db")
+
+
+@terraform("ebs_modify_iops_percent")
+def test_ebs_modify_iops_percent_gp3(test, ebs_modify_iops_percent):
+    session_factory = test.replay_flight_data("test_ebs_modify_iops_percent_gp3")
+    vol_id = ebs_modify_iops_percent["aws_ebs_volume.test_volume.id"]
+
+    policy = test.load_policy(
+        {
+            "name": "ebs-modify-iops-gp3",
+            "resource": "aws.ebs",
+            "filters": ["modifyable", {"VolumeType": "gp3"}],
+            "actions": [
+                {
+                    "type": "modify",
+                    # Increase IOPS by 30% using only iops-percent
+                    "iops-percent": 130,
+                }
+            ],
+        },
+        session_factory=session_factory,
+    )
+
+    resources = policy.run()
+    assert len(resources) == 1
+    assert resources[0]["VolumeId"] == vol_id
+
+    if test.recording:
+        time.sleep(10)
```

Comment:
For future reference, a waiter (specifically https://boto3.amazonaws.com/v1/documentation/api/latest/guide/clients.html#waiters, with overall reference at https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ebs/client/get_waiter.html) is generally preferred to a generic `sleep`. IMO, not a big deal in this case (since it only affects recording), but is a better means of handling eventually-consistent resources.